sourcegraph/cody
# Autocomplete: Very rudimentary Python support for LSP graph context

Closes #1237 

This is very rudimentary, we need a better solution like #1244 

The main motivation is that this will increase the size of the test pool for this feature flag quite a bit 

## Test plan

- Enable LSP graph context
- Click into a Python file
- Observe the hydrated sections in the trace view
- Continue typing and check the context being added
